velox/functions/sparksql/registration/RegisterArray.cpp

```diff
@@ -196,6 +197,11 @@ void registerArrayFunctions(const std::string& prefix) {
       Array<Generic<T1>>,
       Generic<T1>>({prefix + "array_append"});
   registerArrayUnionFunctions(prefix);
+  registerFunction<
```

Comment:
Please add the fast path registration for primitive type as other functions

Reply:
I'm not quite sure about this comment. Does it mean to register like the `array_remove` function?

Reply:
yes

Reply:
ok, udpated :)

Reply:
Please see `callNullable` to replace current function `call`